future_work/README.md

```diff
@@ -0,0 +1,37 @@
+# Future Work
+
+This is a list of things we haven't tried yet and which might be interesting to take a look at:
+
+## CPShell Escape
+
+- Related work: [CVE-2022-3086](https://securitybytes.me/posts/cve-2021-37471/)
+- Enable `su-cmds` in the CPShell command prompt:
+  - Set `superuser` to `True` in `cpshell.pyc` and [rebuild/reflash the rootfs](../README.md#flashing-the-nand-images)
+  - Enable technical support access in `service_manager.pyc` by modifying the `techsupport_access` method and [rebuild/reflash the rootfs](../README.md#flashing-the-nand-images)
+- Try a shell escape with [ssh](https://gtfobins.github.io/gtfobins/ssh/), [telnet](https://gtfobins.github.io/gtfobins/telnet/), [tcpdump](https://gtfobins.github.io/gtfobins/tcpdump/), [find](https://gtfobins.github.io/gtfobins/find/), [diff](https://gtfobins.github.io/gtfobins/diff/) or [grep](https://gtfobins.github.io/gtfobins/grep/)
+- Read password values with `get` command from config file(s)
+- Edit files with the `edit` command --> Add backdoor to `/etc/rc` script
+- In the router's rootfs grep for possible command injection flaws:
+```
+$ grep -iE "os.system|os.popen|commands.getstatusoutput|commands.getoutput|commands.getstatus|subprocess.call|subprocess.run|subprocess.Popen|pty.spawn|execfile|exec|eval"
+```
+
+## Miscellaneous
```

Comment:
what about the USB port?

Reply:
You mean firmware upgrade via USB? USB stack fuzzing?

Reply:
yes, everything related to USB attack surface.

Reply:
Added.